src/styles/shared/body.scss

```diff
@@ -56,6 +65,33 @@ body {
   }
 }
 
+.trailer {
+  padding: 1rem;
+  border: solid 100px #eae2d6;
+  border-radius: 20px;
+  background: url('/src/assets/images/trencadis/very-light.svg') #fbf8f4 center /
+    640px repeat;
+  background-color: $c-during-bg-card;
+  border-image: url('/src/assets/images/borders/crown.svg') 100 repeat;
+
+  @media screen and (max-width: 600px) {
+    border: none;
+  }
+}
+
+.section__video {
+  max-width: calc(100vw - 32px);
+  max-height: calc(calc(100vw - 32px) * 0.5625);
```

Comment:
You can use the aspect-ratio property instead of the max_height

Reply:
changed